cmd/kube-spawn/setup.go

```diff
@@ -43,6 +44,7 @@ func init() {
 
 	cmdSetup.Flags().IntVarP(&numNodes, "nodes", "n", 1, "number of nodes to spawn")
 	cmdSetup.Flags().StringVarP(&baseImage, "image", "i", "", "base image for nodes")
+	cmdSetup.Flags().StringVarP(&kubeSpawnDir, "kube-spawn-dir", "k", "", "path to .kube-spawn directory")
```

Comment:
We already use `-k`  for `--kubernetes-version` [here](https://github.com/kinvolk/kube-spawn/blob/master/cmd/kube-spawn/kube-spawn.go#L53)